ui/apps/everest/src/utils/cron-converter/converter.ts

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { getTimezoneOffset } from 'date-fns-tz';
+import cronParser from 'cron-parser';
+
+const cronConverter = (
+  cronExpression: string,
+  originalTimeZone: string,
+  targetTimeZone: string
+): string => {
+  const interval = cronParser.parseExpression(cronExpression);
+  const now = new Date();
+
+  const timeZoneOffsetMin =
```

Comment:
As we talked, this code was taken from here: https://www.npmjs.com/package/crontzconvert

Just adapted it to date-fns instead of moment. But yes, it's difference in minutes, hence the division